tests/conftest.py

```diff
@@ -21,13 +21,15 @@
 
 
 def pytest_addoption(parser):
-    # the default backend is Speculos, this option allows to select another
     parser.addoption("--backend", action="store", default="speculos")
     parser.addoption("--display", action="store_true", default=False)
+    parser.addoption("--elfs_dir", action="store", default="bin")
     parser.addoption("--golden_run", action="store_true", default=False)
+    parser.addoption("--log_apdu_file", action="store", default=None)
     # Enable using --'device' in the pytest command line to restrict testing to specific devices
     for fw in FIRMWARES:
-        parser.addoption("--"+fw.device, action="store_true", help="run on nanos only")
+        parser.addoption("--"+fw.device, action="store_true", help=f"run on {fw.device} only")
```

Comment:
Wan't `--device {fw.device}` the chosen argument name?

Reply:
This Pr as not be updated.
I will update it once https://github.com/LedgerHQ/ragger/pull/83 is merged.